### DISCUSSION TITLE
USB SSD for Proxmox Backup Server LXC

### DISCUSSION THREAD

**Discussion by markPHL** [tteck/Proxmox#2990] [Help Needed]
Hi all, looking for best practice suggestions for my scenario as I'm seeing some conflicting guidance when I search for similar discussions and resources on the web and YouTube. 

I repurposed an old NUC to run a second Proxmox node. I'd like to use this node to run Proxmox Backup Server as well as have the flexibility to run additional containers like ZwaveJS and Zigbee2MQTT. I have a 500 GB Samsung T7 connected to exclusively dedicate to this Proxmox Backup Server LXC.

What's the best practice for leveraging this storage within Proxmox Backup Server LXC?
- mount on the host and then use mount points/bind mounts to pass into PBS?
- use Device Passthrough to pass the whole device (or port?) into PBS?
- something other approach I'm missing?

Not sure if there are pros/cons of one approach over the other and would love to learn - running PVE 8.2.2 if that matters. Thanks!

Edit - decided I was overthinking things and set up a bind mount, works flawlessly! If that's not the best practice, I'd still love to learn other perspectives. Thanks all!

**Comment by Mellowlynx** [comment]
I have a five NUC' Proxmox cluster and decided early to only get the models with both an M.2 and SATA connector.
(they are in a rack mount, and looks cleaner this way) 📷 Below
The extra disk in on the SATA connection, and I opted to add the full disk as a Directory on the node.

Everything on the node can access it, and by adding a second disk (vm) or mount point (lxc) you can decide if and how to back up the data.
(For example, I have an LXC that syncs the data to an external site, so the second drive is not backed up at all, saving space locally)

Also, it's easier to restore, just recreate the directory and restore the disk files (I have a backup folder on my NAS mounted within the cluster, so all nodes can access it). Because only a view VM/LXC's are using it, it's fine that they will take a bit longer to get back online.

This works for me, and it's fast too, using it as a Nextcloud data drive, and photo's and video's load quick enough.
For others, this might not be an option, the best way, is the way that works for you. 😃
I probably would add the drive as a Directory on the node, and give the backup LXC a second drive and store the backups on there.
If you get another LXC that needs a bit more storage space, you can just add it to the same node as the PBS and give it that space (if you have any left, off course 😉)

Just be sure to test recovery from a backup too! Allot do not do that, but is in my eyes essential.
_What happens when the node with PBS that goes down..._

Hope this might help you a bit, and let me know if something is not clear.

**For who is interested:**
Part of my home lab
![image](https://github.com/tteck/Proxmox/assets/2348339/5c3671ec-c827-4535-a4bb-6ff0a992b452)

**Reply by markPHL** [reply]
That is a great looking setup! Appreciate the tips on how else I can set this up (and to test restoring a backup ... gonna do that now! 😁). I love how there are so many options available.
